hat/assets/js/apps/Iaso/domains/entities/duplicates/details/submissions/SubmissionsForEntity.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FunctionComponent } from 'react';
+import { useSafeIntl } from 'bluesquare-components';
+import { Divider } from '@material-ui/core';
+import { ExpandableItem } from '../../../../app/components/ExpandableItem';
+import { InstanceDetailRaw } from '../../../../instances/compare/components/InstanceDetailRaw';
+import { useGetInstancesForEntity } from '../hooks/useGetInstancesForEntity';
+import MESSAGES from '../../messages';
+import WidgetPaperComponent from '../../../../../components/papers/WidgetPaperComponent';
+import { NoSubmission } from './NoSubmission';
+
+type Props = {
+    entityId?: string;
+    title: string;
+};
+
+export const SubmissionsForEntity: FunctionComponent<Props> = ({
+    entityId,
+    title = 'Entity',
+}) => {
+    const { formatMessage } = useSafeIntl();
+    const {
+        data = { instances: [] },
+        isLoading,
+        isError,
+    } = useGetInstancesForEntity({ entityId });
+
+    return (
+        <WidgetPaperComponent title={title} padded>
+            {data.instances.length === 0 && <NoSubmission />}
+            {data.instances.length > 0 &&
+                data.instances.map((instance, index) => (
+                    <>
+                        {index === 0 && (
+                            <Divider style={{ height: '1px', width: '100%' }} />
```

Comment:
Isn't the `Divider` by default with this style ?

Reply:
Nope, if I don't add it it's invisible